src/QSymbolicsBase/predefined_fock.jl

```diff
@@ -26,6 +26,21 @@ end
 SqueezedState(z::Number) = SqueezedState(z, inf_fock_basis)
 symbollabel(x::SqueezedState) = "0,$(x.z)"
 
+"""Two-mode squeezed vacuum state, or EPR state, in defined Fock basis."""
+@withmetadata struct EPRState <: SpecialKet
```

Comment:
`EPR` might be confused with a qubit Bell pair. How about `TwoSqueezeState` or `CVBellPair` or something like that.